docs/development/windows_support.md

```diff
@@ -126,25 +126,14 @@ These instructions mostly mirror the instructions in the root
 #### Install tools
 
 > [!TIP]
-> These tools are available via package managers like
-> [chocolatey](https://chocolatey.org/):
->
-> ```bash
-> choco install visualstudio2022buildtools -y --params "--add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.VC.CMake.Project --add Microsoft.VisualStudio.Component.VC.ATL --add Microsoft.VisualStudio.Component.Windows11SDK.22621"
-> choco install git.install -y --params "'/GitAndUnixToolsOnPath'"
-> choco install cmake --version=3.31.0 -y
-> choco install ninja -y
-> choco install ccache -y
-> choco install python -y
-> choco install strawberryperl -y
-> choco install pkgconfiglite -y
-> ```
-
-> [!TIP]
-> winget installation is recommended for dvc
-> [winget-cli](https://github.com/microsoft/winget-cli):
->
+> These tools are available via package managers like winget on Windows:
```

Comment:
mdformat from pre-commit wants an extra newline here before the code block: https://github.com/ROCm/TheRock/actions/runs/20382736872/job/58576893829?pr=2626#step:4:201
```suggestion
> These tools are available via package managers like winget on Windows:
>
```

We can also add pre-commit to our [`requirements.txt`](https://github.com/ROCm/TheRock/blob/main/requirements.txt), that file is for packages needed for development. I typically install pre-commit system-wide and use it across multiple projects but it's fine to also have it in the build venv.